Bug fix hot reloading

coopy opened the pull request:
This, along with an [update to each victory component](https://github.com/FormidableLabs/victory/issues/165), will make hot reloading work.

Fixes https://github.com/FormidableLabs/builder-victory-component/issues/9

Commit:
Use only-dev-server to not reload on syntax errors

Commit:
Upgrade react-hot-loader

Commit:
Use --inline option in hot webpack config

This removes the need for a <script src="webpack-dev-server">

knowbody commented:
@per as react-hot-loader is deprecated, maybe we could move to react transform in this PR?

coopy commented:
That's a bigger project, and I want to land this soon. But please feel free
to ticket it out!
tors 14 jan. 2016 kl. 03:31 skrev Mateusz Zatorski <notifications@github.com

> :
> 
> @per https://github.com/per as react-hot-loader is deprecated, maybe we
> could move to react transform in this PR?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/FormidableLabs/builder-victory-component/pull/25#issuecomment-171618427
> .

knowbody commented:
will do, thanks for this PR!

Commit:
Remove NoErrorsPlugin

No longer needed with react-hot-loader upgrade:
https://github.com/erikras/react-redux-universal-hot-example/issues/147
